statistics/handle/cache/statscacheinner.go

```diff
@@ -101,8 +106,13 @@ func (sc *StatsCache) Values() []*statistics.Table {
 }
 
 // FreshMemUsage refreshes the memory usage of the cache.
+// Values in StatsCache should be read-only, but when initializing the cache, some values can be modified.
+// To make the memory cost more accurate, we need to refresh the memory usage of the cache.
 func (sc *StatsCache) FreshMemUsage() {
-	sc.c.FreshMemUsage()
+	values := sc.c.Values()
+	for _, v := range values {
+		sc.c.Put(v.PhysicalID, v)
```

Comment:
For LRU, Put will change the sequence of LRU, FreshMemUsage should not have that side-effect.

Reply:
This method `FreshMemUsage` is only used when initializing, so the sequence change is acceptable.

Reply:
What about keeping the previous implementation and renaming it to InitMemoryUsage since it's only used when initializing? I'm afraid the new implementation will make init longer than before when there are many tables?